CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Contributing Guidelines
```

Comment:
Where did this come from? I'm not sure I dislike it but it's pretty long and I'm not sure all of it applies to this project, might be worth an edit, or checking the Nexmo/Vonage standard examples https://github.com/Nexmo/repo-standards

Reply:
I may have borrowed it from repo we have. But I'm afraid I can't remember which one!